t/plugin/forward-auth.t

```diff
@@ -143,6 +143,24 @@ property "request_method" validation failed: matches none of the enum values
                                                end
                                            end
                                         end
+                                    end]],
+                                    [[return function(conf, ctx)
```

Comment:
please write the test cases in a new file `t/plugin/forward-auth2.t` . This file already contains too many serverless functions and routes which can cause conflict.

Reply:
@shreemaan-abhishek Had split the testcases into new file.